annotations/fenix/metrics/top_sites.open_baidu_search_attribution/README.md

```diff
@@ -0,0 +1,8 @@
+---
+tags:
+  - Tabs
```

Comment:
I'm not an expert, but wouldn't this fall under "Search"?

```suggestion
  - Search
```

Reply:
@wlach Hi, this metric is actually a record of opening a topsite of Baidu, just like this (Google) https://github.com/mozilla/glean-annotations/tree/main/annotations/fenix/metrics/top_sites.open_google_search_attribution

Reply:
if we couldn't add top site category I will lean more towards the search one.

Reply:
There is in fact a `TopSites` category 😄 

I think we should probably add `Search` as well though, since this will be relevant to people who care about that (e.g. Revenue Data Science!).

Reply:
I made the proposed modifications for `top_sites.open_google_search_attribution` in #68

Reply:
👍🏽 , I already ✅  the PR :)